CleverRaven/Cataclysm-DDA
# sidebar wielded gun fire modes fix

#### Summary
Bugfixes "Make active fire modes from attached gunmods appear in the sidebar."


#### Purpose of change
_Should_ fix #53864


#### Describe the solution
Changed a conditional "not thing or thing" to "not thing and not thing" in the code


#### Describe alternatives you've considered



#### Testing
1. Wield a gun with an underslung shotgun attached. 
2. Switch the firing mode of the gun to use the underslung shotgun
3. See if the current firing mode is correctly shown in the side bar (in the "wielded:" widget)


#### Additional context
I need someone to sanity check this for me, I'm not entirely sure if this part of the code works the way I think it does.
